lib/index.js

```diff
@@ -1,5 +1,9 @@
 /* eslint no-process-env: 0, no-console: 0, max-statements: 0 */
 'use strict';
+process.on('unhandledRejection', function(e) {
+  // no swallowed Promise errors
+  throw e;
```

Comment:
Yeah, cases like that is where Bluebird's `.asCallback()` is useful for interop; this is subtly broken, and it's not apparent how. I'm glad you're adding something to help us potentially catch stuff like this.